# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webrtc/transport/test/webrtcproxychannel_unittest.cpp
# Commit: 8e7c97a37073
# Full Hash: 8e7c97a37073a14ba832e2f5c27f2c11b7787ef5
# Author: Valentin Gosu <valentin.gosu@gmail.com>
# Date: 2023-03-15 16:26:49
# Regressor Bug: 1805372
# File Overlap Count: 1
# Description:
#   Bug 1805372 - Pass effectiveTRRMode to nsHttpChannel from socketTransport/DnsAndConnectSocket r=necko-reviewers,kershaw
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D164539
# ==============================================================================

diff -r 5abfd5ad2e35 -r 8e7c97a37073 dom/media/webrtc/transport/test/webrtcproxychannel_unittest.cpp
--- a/dom/media/webrtc/transport/test/webrtcproxychannel_unittest.cpp	Wed Mar 15 12:41:26 2023 +0000
+++ b/dom/media/webrtc/transport/test/webrtcproxychannel_unittest.cpp	Wed Mar 15 13:00:46 2023 +0000
@@ -185,6 +185,15 @@
     MOZ_ASSERT(false);
     return NS_OK;
   }
+  NS_IMETHOD GetEffectiveTRRMode(
+      nsIRequest::TRRMode* aEffectiveTRRMode) override {
+    MOZ_ASSERT(false);
+    return NS_OK;
+  }
+  NS_IMETHOD GetTrrSkipReason(nsITRRSkipReason::value* aSkipReason) override {
+    MOZ_ASSERT(false);
+    return NS_OK;
+  }
   NS_IMETHOD GetRetryDnsIfPossible(bool* aRetryDns) override {
     MOZ_ASSERT(false);
     return NS_OK;